requirements/ci/nox.lock/py310-linux-64.lock

```diff
@@ -235,6 +238,7 @@ https://conda.anaconda.org/conda-forge/noarch/pyopenssl-22.0.0-pyhd8ed1ab_0.tar.
 https://conda.anaconda.org/conda-forge/linux-64/pyqt5-sip-12.9.0-py310hd8f1fbe_1.tar.bz2#21ae1ac216cfc85e24dc7c09570ddf31
 https://conda.anaconda.org/conda-forge/noarch/pytest-forked-1.4.0-pyhd8ed1ab_0.tar.bz2#95286e05a617de9ebfe3246cecbfb72f
 https://conda.anaconda.org/conda-forge/linux-64/qt-main-5.15.4-ha5833f6_2.tar.bz2#dd3aa6715b9e9efaf842febf18ce4261
+https://conda.anaconda.org/conda-forge/noarch/setuptools_scm-7.0.4-hd8ed1ab_0.tar.bz2#b068a47ce183af12a53c6b0d84f96085
```

Comment:
Why two versions of `setuptools-scm` in the lock files?

Reply:
Yeah, noticed that... I think it's basically because they have two names for the same thing i.e, `setuptools-scm` and `setuptools_scm`.

Perhaps `setuptools_scm` is aliased to `setuptools-scm`... I'll dig and see what I find out.

Reply:
As suspected, `setuptools_scm` is an alias package to `setuptools-scm`, so I've dealt with that now :+1: